browser/brave_vpn/android/brave_vpn_native_worker.cc

```diff
@@ -127,6 +127,89 @@ void BraveVpnNativeWorker::OnGetProfileCredentials(
       success);
 }
 
+void BraveVpnNativeWorker::GetWireguardProfileCredentials(
```

Comment:
we should be using mojo for this stuff now. This usage of JNI is deprecated

Reply:
@bridiver i will be addressing mojo change as a follow up as it's an old PR we should merge first.